### ISSUE TITLE
Empty index.d.ts and index.metadata.json cause build error

### ISSUE THREAD

**Issue by vincent-cm** [webcat12345/ngx-ui-switch#273]
ERROR in Unexpected end of JSON input
when building with AoT: ng build --prod

**Comment by cmckni3** [comment]
Which versions of Angular and `ngx-ui-switch`?

**Comment by vincent-cm** [comment]
@cmckni3 

ngx-ui-switch 6.0.2
@angular/core ^6.0.3
node v10.3.0
npm 6.1.0

**Comment by cmckni3** [comment]
Fixed in `ngx-ui-switch@6.0.3`. The package wasn't published correctly due the `publish` command being broken in yarn `1.7.0`

**Comment by cmckni3** [comment]
This was actually an issue with `npmjs.org`

https://status.npmjs.org/

>>>
Jun 1, 2018
Missing package versions after apparently successful publishes
Resolved - This incident has been resolved.
Jun 1, 17:06 UTC
Monitoring - We have identified the issue and submitted a fix. We will continue to monitor the situation.
May 31, 17:19 UTC
Investigating - We are investigating reports of missing package versions after what appears like a successful publish from the CLI.
May 31, 16:26 UTC
